Sources/Phakchi/Encodable.swift

```diff
@@ -1,52 +1,63 @@
 import Foundation
 
-typealias JSONObject = [String: AnyObject]
+typealias JSONObject = [String: PactEncodable]
 
 public protocol PactEncodable {
-    var pactJSON: AnyObject { get }
+    var pactJSON: PactEncodable { get }
```

Comment:
`pactJSON` should, as it name says, return JSON, not something that is encodable to JSON!?